docs/en/observability/apm/feature-roles.asciidoc

```diff
@@ -90,23 +133,88 @@ that has the following privileges:
 |Index
 |`create_doc` on `traces-apm*`, `logs-apm*`, and `metrics-apm*` indices
 |Write events into {es}
+
+|Cluster
+|`monitor`
+a|* Allows cluster UUID checks, which are performed as part of APM server startup preconditions
+if {ref}/security-settings.html[Elasticsearch security] is enabled (it is enabled by default).
+* Allows a license check, which is required if <<apm-tail-based-sampling,tail-based sampling>> is enabled.
+
+NOTE: If you have explicitly disabled Elastic security _and_ you are _not_ using tail-based sampling,
+this privilege may not be necessary.
 |====
 
-. Assign the *general writer role* to users who need to publish APM data.
+. Assign the *general writer role* to APM Server users who need to publish APM data.
 
-. If <<apm-configuration-rum,real user monitoring>> is enabled, create a separate <<apm-privileges-rum-source-mapping,RUM source mapping role>>.
+[NOTE]
+====
+Assign <<apm-feature-roles,additional APM feature roles>> to users as needed including the
+_Central configuration management role_, which is <<apm-central-config-role-note,required in most cases>>.
+====
 
 ////
 ***********************************  ***********************************
 ***********************************  ***********************************
 ////
 
-[[apm-privileges-to-publish-monitoring]]
-=== Grant privileges and roles needed for monitoring
+[float]
+[[apm-privileges-agent-central-config]]
+=== Create a _central config_ role
 
-++++
-<titleabbrev>Create a _monitoring_ user</titleabbrev>
-++++
+[[apm-central-config-role-note]]
+[IMPORTANT]
+====
+The privileges included in this role are *required* for all users when <<apm-agent-configuration,central configuration management>> is enabled (it is enabled by default). You do _not_ need the _Central configuration management_ role only if central configuration management has been explicitly disabled in the APM UI.
+====
+
+[[apm-privileges-agent-central-config-server]]
+APM Server acts as a proxy between your APM agents and the APM UI.
+The APM UI communicates any changed settings to APM Server so that your agents only need to poll the Server
+to determine which central configuration settings have changed.
+
+To grant an APM Server user with the required privileges for managing central configuration in {es} without {kib},
+assign the user the following privileges:
+
+[options="header"]
+|====
+|Type | Privilege | Purpose
+
+| Index
+|`read` on `.apm-agent-configuration` index, `allow_restricted_indices: true`
+|Allow APM Server to manage central configurations in {es}
```

Comment:
@endorama to confirm what @carsonip said in https://github.com/elastic/observability-docs/pull/4193#issuecomment-2374558159. Let me know if this needs to change or if we need to provide more information about how to configure this.

Reply:
Permissions are correct, but we need to mention that this cannot be achieved with the UI. Linking to https://www.elastic.co/guide/en/observability/current/apm-beats-api-keys.html would provide the appropriate additional information. Incidentally there is an open PR related to that page, as currently does not show the correct information https://github.com/elastic/observability-docs/pull/4083

Reply:
I understand that we can't configure this role in the UI, but I'm confused about why we would point to the [Grant access using API keys](https://observability-docs_bk_4083.docs-preview.app.elstc.co/guide/en/observability/master/apm-beats-api-keys.html) page. Unless I'm misunderstanding, _Grant access using API keys_ describes an _alternative_ to creating roles, not how to use an API to create/update roles. Is that correct? Are we saying that the user shouldn't use a role to give an APM Server user this kind of access? Or do we want to instruct the user to use an API ([Create or update roles API](https://www.elastic.co/guide/en/elasticsearch/reference/current/security-api-put-role.html)?) to create the _central config_ role?

Reply:
Would something like this be appropriate? 

>To create a role with the required privileges for managing central configuration in Elasticsearch without Kibana, you must to use the [Roles API](https://www.elastic.co/guide/en/elasticsearch/reference/current/security-api-put-role.html) (the required privileges can't be assigned to a role in Kibana):
>
> ```
>POST /_security/role/apm_agentcfg
>{
>  "description": "Allow APM Server to manage central configurations in Elasticsearch.",
>  "indices": [
>    {
>      "names": [".apm-agent-configuration"],
>      "privileges": ["read"],
>      "allow_restricted_indices": true
>    }
>  ]
>}
>```

Reply:
> I understand that we can't configure this role in the UI, but I'm confused about why we would point to the [Grant access using API keys](https://observability-docs_bk_4193.docs-preview.app.elstc.co/guide/en/observability/master/apm-beats-api-keys.html) page. Unless I'm misunderstanding, Grant access using API keys describes an alternative to creating roles, not how to use an API to create/update roles. 

Yes this is correct and yes also to your confusion! 

I think your proposal to use the Roles API is better in this context! I haven't tested it yet but looking at the permissions we grant to the API key and the Roles API docs everything seems correct. I'll test it tomorrow.

Reply:
tested and it's working correctly! The user with that role assigned was correctly able to read the APM agent configurations, while removing the role produced an error. :sparkles: